src/workloads/query/CPUCycleMetricsUpdate.yml

```diff
@@ -0,0 +1,63 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sys-perf"
+Description: |
+  This workload is designed to insert 10k documents into a single collection and then
+  perform exactly 10k reads. This is designed to help us calculate CPU cycle metrics
+  when utilizing the Linux 3-Node ReplSet CPU Cycle Metrics 2023-06 variant
+
+dbname: &dbname 10kDocs
+DocumentCount: &NumDocs 10000
+CollectionCount: &NumColls 1
+
+NumPhases: &NumPhases 2
+
+Document: &doc
+  a: {^Inc : {}}
+  b: {^RandomString: {length: 16}}
+
+Actors:
+  - Name: Setup
+    Type: Loader
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [0]
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          # Insert 10k documents to our singular collection
+          Repeat: 1
+          BatchSize: 100
+          Threads: 1
+          DocumentCount: *NumDocs
+          Database: *dbname
+          CollectionCount: *NumColls
+          Document: *doc
+          Indexes:
+          - keys: {a: 1}
+
+  - Name: UpdateActor
+    Type: CrudActor
+    Threads: 1
+    Database: *dbname
+    Phases:
+      OnlyActiveInPhases:
+        Active: [1]
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Repeat: 10000
+          RecordFailure: true
+          ThrowOnFailure: true
+          CollectionCount: *NumColls
+          MetricsName: "load"
+          Operations:
+          - OperationName: updateOne
+            OperationMetricsName: updates
+            OperationCommand:
+                Filter: { a: { ^RandomInt: { min: 0, max: *NumDocs } } }
+                Update: { $set: { ^RandomInt: { min: 0, max: *NumDocs } } }
```

Comment:
We might want to update b instead of a.

Also, do we want to use inc to filter for an item?